demo/api/src/app.module.ts

```diff
@@ -159,9 +160,11 @@ export class AppModule {
                     ? [
                           ContentGenerationModule.register({
                               Service: ContentGenerationService,
-                              config: {
-                                  openAiContentGeneration: config.contentGeneration,
-                              },
+                              imports: [
+                                  AzureOpenAiContentGenerationModule.register({
+                                      config: config.contentGeneration,
```

Comment:
Do we need this nested `config`?

Reply:
No: https://github.com/vivid-planet/comet/pull/1712/commits/b0ad6c98637c9dcf68eb19f456395dd4c5ba70f8